code/modules/hivemind/mobs.dm

```diff
@@ -0,0 +1,913 @@
+//High chance of malfunction
+//Default speaking chance
+//Appears from dead cyborgs
+//////////////////////////////////////////////////////////////////////////////
+
+/mob/living/simple_animal/hostile/hivemind/hiborg
+	name = "cyborg"
+	desc = "This cyborg is covered with something, seems like it's 'something' is alive."
+	icon_state = "hiborg"
+	icon_dead = "hiborg-dead"
+	health = 220
+	maxHealth = 220
+	melee_damage_lower = 10
+	melee_damage_upper = 15
+	attacktext = "punches with claw"
+	speed = 12
+	malfunction_chance = 15
+	mob_size = MOB_MEDIUM
+
+	speak = list("Everytime something breaks apart. Hell, i hate this job!",
+				"What? I hear something. Just mice? Just mice, phew...",
+				"I'm too tired, man, too tired. This job is... Awful.",
+				"These people knows nothing about this work, and about me. I can surprise them.",
+				"Blue wire is bolts, green is safety. Just... Pulse it here, okay? Right...")
+	target_speak = list(
+						"I know what's wrong, just give me fix that.",
+						"You need my help? What's wrong? Gimme that thing, i can fix that.",
+						"Si-i-ir... Sir. Sir. It's better to... Stop here! Stop i said, what are you!?",
+						"Wait! Hey! Can i fix that!? I'm an engineer, you fuck! Sto-op-op-p here, i know what to do!"
+						)
+
+
+/mob/living/simple_animal/hostile/hivemind/hiborg/AttackingTarget()
+	if(!Adjacent(target_mob))
+		return
+
+	//special attacks
+	if(prob(10))
+		splash_slash()
+		return
+
+	if(prob(40))
+		stun_with_claw()
+		return
+
+	return ..() //default attack
+
+
+/mob/living/simple_animal/hostile/hivemind/hiborg/proc/splash_slash()
+	src.visible_message(SPAN_DANGER("[src] spins out the place and slash everyone in a circle!"))
+	for(var/atom/target in range(1, src))
+		if(target != src)
+			target.attack_generic(src, rand(melee_damage_lower, melee_damage_upper*2))
+	if(!client && prob(speak_chance))
+		say(pick("Get away from me!", "They are everywhere!"))
+
+
+/mob/living/simple_animal/hostile/hivemind/hiborg/proc/stun_with_claw()
+	if(isliving(target_mob))
+		var/mob/living/victim = target_mob
+		victim.Weaken(5)
+		src.visible_message(SPAN_WARNING("[src] holds down [victim] to the floor with his claw."))
+		if(!client && prob(speak_chance))
+			say(pick("Stand still, i'll make it fast!",
+					"I will fix you! Don't resist! Don't resist you rat!",
+					"I want just replace that broken thing!"))
+
+
+/////////////////////////////////////HIMAN////////////////////////////////////
+//Hive + Man
+//Special ability: Shriek, that stuns victims
+//Can fool his enemies and pretend to be dead
+//A little bit higher chance of malfunction
+//Default speaking chance
+//Appears from dead human corpses
+//////////////////////////////////////////////////////////////////////////////
+
+/mob/living/simple_animal/hostile/hivemind/himan
+	name = "human"
+	desc = "This guy is totally not human. You can see tubes across of his body and some kind metal limbs."
```

Comment:
This guy is totally not human. You can see tubes all across his body and metal where flesh should be.